common/src/com/thoughtworks/go/serverhealth/ServerHealthService.java

```diff
@@ -72,6 +73,25 @@ public HealthStateType update(ServerHealthState serverHealthState) {
         }
     }
 
+    // called from spring timer
+    public synchronized void onTimer(CruiseConfigProvider provider) {
+        CruiseConfig currentConfig = provider.getCurrentConfig();
+        getAllValidLogs(currentConfig);
+        LOG.info("Recomputing material to pipeline mappings.");
```

Comment:
Should this be INFO level or DEBUG? I guess it's ok because our default log level for the appender is WARN.